src/auth/components/forms/twitter-email/twitter-email.html

```diff
@@ -26,6 +26,12 @@
 		>
 		auth.linked_account.twitter.finalize.duplicate_email_error_html
 	</div>
+	<div class="alert alert-warning anim-fade-in-enlarge no-animate-leave"
```

Comment:
Instead of reporting the error here - consider doing the rate limit at an earlier stage to make this the same flow as the other auth operations.